src/dataframe/dataframe.jl

```diff
@@ -972,11 +972,15 @@ disallowmissing!(df::DataFrame, cols::Colon=:) =
                  compress::Bool=false)
     categorical!(df::DataFrame, cnames::Union{Regex, Not};
                  compress::Bool=false)
-    categorical!(df::DataFrame; compress::Bool=false)
+    categorical!(df::DataFrame, cols::Type=Union{AbstractString, Missing};
```

Comment:
Maybe put this one first since it's the simplest method?

Reply:
done